ui/src/containers/Resources/index.tsx

```diff
@@ -72,10 +76,30 @@ const Resources: React.FC = observer(() => {
     );
   };
 
-  return resources.resources.size === 0 ? (
-    <Spinner className="hub-spinner" />
-  ) : (
-    <React.Fragment>{checkResources(resources.filteredResources)}</React.Fragment>
-  );
+  const checkResourceStatus = (resources: IResourceStore) => {
+    return resources.err === '' ? (
+      resources.resources.size === 0 && resources.status !== 200 && resources.status !== 404 ? (
+        <Spinner className="hub-spinner" />
+      ) : (
+        <React.Fragment>{checkResources(resources.filteredResources)}</React.Fragment>
+      )
+    ) : (
+      <EmptyState variant={EmptyStateVariant.large} className="hub-resource-emptystate__margin">
```

Comment:
Can we have a better class name instead of `className="hub-resource-emptystate__margin`